server/odc-service/src/main/java/com/oceanbase/odc/service/connection/ConnectionValidator.java

```diff
@@ -74,4 +94,17 @@ void validatePrivateConnectionTempOnly(Boolean temp) {
                     "Cannot create persistent connection due temp only for private connection");
         }
     }
+
+    void validateProjectOperable(Long projectId) {
```

Comment:
The method name is a bit vague and is it appropriate to replace this method in the `ConnectionValidator`?

Reply:
I mean this method is related to project level authentication, why do we place it in `ConnectionValidator` which seems like a connection-level thing?

Reply:
I got you!